code/_onclick/ai.dm

```diff
@@ -33,9 +33,9 @@
 
 	if(control_disabled || stat)
 		return
-		
+
 	var/turf/pixel_turf = get_turf_pixel(A)
```

Comment:
there is a diffence between a feature port. and someone suggesting some nice code you can use that could make the code more accurate. The code could have come from hippie for all i care if it works and its more accurate then what we currently have i would use it. There is NOTHING wrong with that.